docs/rootfs-and-kernel-setup.md

```diff
@@ -119,8 +119,14 @@ Alpine Linux:
 
    # Then, copy the newly configured system to the rootfs image:
    for d in bin etc lib root sbin usr; do tar c "/$d" | tar x -C /my-rootfs; done
+   
+    # This command may trigger the following messages
+   ```bash
```

Comment:
this is already inside a code block.
Please reformat the message and make sure that the markdown rendering is easy to read